frontend/src/components/VMediaInfo/VCollapsibleTagSection.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <div class="-my-1.5px">
+    <ul
+      ref="tagsContainerRef"
+      class="flex flex-wrap gap-3 overflow-y-hidden p-1.5px"
+      :class="heightClass"
+    >
+      <li v-for="tag in visibleTags" :key="tag">
+        <VTag :href="localizedTagPath(tag)">{{ tag }}</VTag>
+      </li>
```

Comment:
Since we have `accuracy` and `provider` information available on the tags, one of the thoughts I had when thinking about how these results would manifest in the frontend was how we might be able to convey that information too. I [mentioned it in the initial design issue](https://github.com/WordPress/openverse/issues/4192#issuecomment-2077689928), but I'm wondering - could the individual tags (at least for the generated section) maybe also have `Source: <provider>, Accuracy: <accuracy>%` tooltips available on them?

Reply:
I like the idea of a tooltip. That should probably be decided on in the IP though 🤔

This PR is simple enough as is, and improves the _current_ frontend (we disclose the clarifai tags) that maybe we should get it merged as is? Alternatively, we draft it and wait for the IP.

Reply:
I think drafting and waiting for the IP is the best move IMO!

Reply:
I don't fully understand the idea. Would you mind sharing an example or making a quick draw?

Reply:
Forgive the poor design quality, but this is the sort of thing I had in mind when I wrote this comment: 

![tag_tooltip](https://github.com/WordPress/openverse/assets/10214785/08b5b9c6-b9cb-48b5-a041-ad69e78157dd)

I imagine there might be some way to include this information in an accessible way, like perhaps as part of the item's label? But this felt like a good way to still surface that information without visual overload.

Reply:
Very clear mockup. Thanks @AetherUnbound

Reply:
@fcoveram could you turn this idea into a new issue with a refined mockup?